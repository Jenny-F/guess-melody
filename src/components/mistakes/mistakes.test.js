import React from "react";
import renderer from "react-test-renderer";
import Mistakes from "./mistakes.jsx";

it(`Should Mistakes component render correctly`, () => {
  const tree = renderer.create(
      <Mistakes
        errorsCount={2}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
