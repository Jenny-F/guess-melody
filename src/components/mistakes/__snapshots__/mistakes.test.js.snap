// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Mistakes component render correctly 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
